binary/bitstring.rkt

```diff
@@ -147,6 +148,10 @@
     (define padded-bytes (bytes->immutable-bytes mutable-padded-bytes))
     (constructor:bitstring padded-bytes padding)))
 
+(define (sequence->bitstring seq)
+  (for/bitstring ([bit seq])
+    bit))
```

Comment:
Bitstrings are also sequences, so `sequence->bitstring` should avoid reconverting them.
```suggestion
  (if (bitstring? seq)
      seq
      (for/bitstring ([bit seq])
        bit))
```